conanfile_rtc.py

```diff
@@ -9,7 +9,7 @@ class GSLConan(ConanFile):
     description = "Guidelines Support Library"
 
     # RTC specific triple
-    settings = "platform_architecture_target"
+    settings = "os"
```

Comment:
The comment also needs an update https://github.com/Esri/entt/blob/runtimecore/conanfile_rtc.py#L13